tokens/models/tokens/base/tokens_base_erc20.sql

```diff
@@ -229,4 +229,5 @@ FROM (VALUES
     , (0xf899b0906f43ea1381e2c13868ebf130edb6c231, 'UNISWAP', 18)
     , (0x9af23c8773adc2a899cba10b60718af3cf06c647, 'FAIRINU', 18)
     , (0x11c1879227d463b60db18c17c20ae739ae8e961a, 'axlBAL', 18)
+    , (0x2dc5dd89a3662567b78fc3a78e1e2c81d9e4d419, 'BANANIA', 18)    
```

Comment:
can confirm this one is included now (after refresh on `tokens.erc20` spell)
```
select *
from tokens.erc20
where contract_address in (
0x2dc5dd89a3662567b78fc3a78e1e2c81d9e4d419
, 0xfc2e6e6bcbd49ccf3a5f029c79984372dcbfe527
, 0x87A1fdc4C726c459f597282be639a045062c0E46
, 0x2dCa80061632f3F87c9cA28364d1d0c30cD79a19
)
```

Reply:
can you remove from PR, to keep file clean?

Reply:
actually, since i'm already in here and merging some PRs, i just did real quick 🙏